### PR TITLE
Added support for transforming comma list values into a boolean value.

### PR DESCRIPTION
**Pull request by michael-mclawhorn**
First part of implementing tag support.

Create an extraction function to transform comma list values to Yes/No

### CODE REVIEW

File: fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactory.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Verizon Media Group
```
Comment (will-lauer) [inline fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactory.java]: My understanding was that this was not the correct copyright to use yet for our code. You may want to check this with Gil before committing it.

File: fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactorySpec.groovy [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        String patternString = String.format(TagExtractionFunctionFactory.DEFAULT_TAG_REG_EX_FORMAT, expression);
+        Pattern.compile(patternString).matcher(value).matches() == expected
+
+        where:
+        value      | expression | expected
+        "11,12,13" | "11"       | true
+        "11"       | "11"       | true
+        "10,11"    | "11"       | true
+        "111"      | "11"       | false
+        ""         | ""         | true
```
Comment (will-lauer) [inline fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactorySpec.groovy]: This last case looks incorrect. We should NOT find 11 when the initial value is empty!

Comment (michael-mclawhorn) [reply fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactorySpec.groovy]: This test case changes the value being sought to emptystring.

Comment (michael-mclawhorn) [reply fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactorySpec.groovy]: On further discussion having a null string tag is probably unreasonable.

File: fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactorySpec.groovy [hunk 3]
```diff
@@ -61,6 +73,5 @@ class TagExtractionFunctionFactorySpec extends Specification {
         "11"       | "11"       | true
         "10,11"    | "11"       | true
         "111"      | "11"       | false
-        ""         | ""         | true
```
Comment (will-lauer) [inline fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/dimension/extractionfunction/TagExtractionFunctionFactorySpec.groovy]: I think its worth having a test of `"" | "11" | false`

**Pull request merged**